helix-core/src/test/java/org/apache/helix/integration/task/TestUpdateWorkflow.java

```diff
@@ -40,7 +40,7 @@
   private static final Logger LOG = LoggerFactory.getLogger(TestUpdateWorkflow.class);
 
   @Test
-  public void testUpdateRunningQueue() throws InterruptedException {
+  public void testUpdateRunningQueue() throws Exception {
```

Comment:
Just curious, why change the InterruptedException to Exception?

Reply:
Using TestHelper.verify throws Exception. So InterruptedException needs to be changed to Exception.